pkg/users/usersDelete.go

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package users
+
+import (
+	"context"
+	"log"
+	"net/http"
+
+	"github.com/galasa-dev/cli/pkg/embedded"
+	galasaErrors "github.com/galasa-dev/cli/pkg/errors"
+	"github.com/galasa-dev/cli/pkg/galasaapi"
+	"github.com/galasa-dev/cli/pkg/spi"
+)
+
+func DeleteUser(loginId string, apiClient *galasaapi.APIClient, console spi.Console) error {
+
+	userData, err := getUserDataFromRestApi(loginId, apiClient)
+
+	if err == nil {
+		err = deletUserFromRestApi(userData, apiClient)
+	}
+
+	return err
+
+}
+
+func deletUserFromRestApi(
+	users []galasaapi.UserData,
```

Comment:
The method deletes a single user, but is being passed multiple users?

Maybe pass a single user to avoid having to check the returned users within this function?